src/pages/urlbox.astro

```diff
@@ -0,0 +1,27 @@
+---
+// npm install urlbox --save
+
+import Urlbox from "urlbox";
+
+
+const { Urlbox_Pub_Key } = import.meta.env;
```

Comment:
I don't think Vite allows for deconstructions of env variables. You have to do `const Urlbox_Pub_Key = import.meta.env.Urlbox_Pub_Key`.

But I would go with the standard uppercase so: `const Urlbox_Pub_Key = import.meta.env.URLBOX_PUB_KEY`